README.md

```diff
@@ -1,2 +1,36 @@
-# dark-theme
-Jenkins Dark Theme (Incubated project)
+# Dark Theme for Jenkins (Incubated project)
+
+[![Gitter](https://badges.gitter.im/jenkinsci/hackfest.svg)](https://gitter.im/jenkinsci/hackfest?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
would it make sense for this to go to ux-sig?

Reply:
I suggest changing it after the hackfest